sunpy/io/special/srs.py

```diff
@@ -161,6 +160,8 @@ def get_meta_data(header):
 
     meta_data = {}
     for m in meta_lines:
+        if "Corrected Copy" in m:
+            continue
```

Comment:
Just do something simple like this?
```suggestion
            meta_data['corrected'] = True
```